org.gnomeradio.Radio.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "44",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=wayland",
+      "--share=ipc",
+  ],
+    "modules": [
+	"shared-modules/intltool/intltool.json",
+	{
+            "name": "cogl",
```

Comment:
this is included in the clutter shared-module

Reply:
OK, should I remove the shared-modules/intltool reference?  It failed to build last time I did this and therefore I opened a new PR, based on the advice I got then.

Reply:
Maybe following the instructions? https://github.com/flathub/shared-modules/

Reply:
You never added the submodule as instructed at the link above.

Reply:
and you can remove this module build in `"shared-modules/clutter/clutter.json"`